qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy

```diff
@@ -221,8 +232,11 @@ class ProcessBaselinesTest extends BaseSpecification {
         sleep 5000
         orchestrator.execInContainer(deployment, "pwd")
 
+        // Give the process indicators time to roll off the queue
+        sleep 60000
```

Comment:
(optional) Would be cool to replace fixed sleep with evaluation whether the expected event has happened

Reply:
It could be included in `waitForViolation`

Reply:
I agree.  I will add that to https://issues.redhat.com/browse/ROX-10903 when I make similar updates to the test I previously ignored.  That way I can get out of the way of these failing nightlies.